### ISSUE TITLE
SplitButton – Target Size (Minimum) (AA)

### ISSUE THREAD

**Issue by gouttierre** [microsoft/fluentui#33650]
### Problem
- Both the medium and small variants fail for the menu button in the horizontal axis. The medium variant fails by only 1px, which should be easy to fix. The small variant fails by 9px.

---
### Current State

- small: 15x24 ❌
- med: 23x32 ❌
- lg: 31x40 ✅ (but should be 32x40)

_[Components / Button / SplitButton - Docs ⋅ Storybook](https://react.fluentui.dev/?path=/docs/components-button-splitbutton--docs#size)_

----
### Change Needed

- small: 24x24 
- med: 24x32 
- lg: 32x40

_[Refer to this Figma UI Kits](https://www.figma.com/design/GvIcCw0tWaJVDSWD4f1OIW/Fluent-2-web?m=auto&node-id=68019-166407&t=mIq4YvlXW0n5rT08-1)_

**Comment by ValentinaKozlova** [comment]
Hi @gouttierre,
This bug has been already fixed (small and medium sizes): https://github.com/microsoft/fluentui/pull/32309
Small:
![Image](https://github.com/user-attachments/assets/8475291e-97c0-4309-9500-c962ea1937ad)

Medium:
![Image](https://github.com/user-attachments/assets/1c73e76a-eec6-4aae-b2ad-667bb1afc6c1)

**Comment by toshie-rashtavie** [comment]
Hi all :) +1 to Valentyna's comment, yep these have been fixed! The 31px is because we'd need to add equal padding on both sides so we can't get the 32px width. I think it's good to have it at 31px!

**Comment by ValentinaKozlova** [comment]
Closing this issue, as no changes needed
